src/components/UserCreatedBounties/BountiesListing/BountiesTabItems.tsx

```
@@ -8,14 +8,18 @@ import { Tabs } from '~src/ui-components/Tabs';
 import BountiesTable from './BountiesTable';
 import { EUserCreatedBountiesStatuses, IUserCreatedBounty } from '~src/types';
 import { useRouter } from 'next/router';
-
+import Image from 'next/image';
+import Link from 'next/link';
```

ESLint

[error] 'Link' is defined but never used.

(@typescript-eslint/no-unused-vars)